emission/net/api/stats.py

```diff
@@ -59,10 +59,16 @@ def storeClientEntry(user, key, ts, reading, metadata):
 
   try:
     response = get_client_stats_db().insert(currEntry)
+    if response == None:
+      get_client_stats_db_backup().insert(currEntry)
   except Exception as e:
     logging.debug("failed to store client entry for user %s, key %s at timestamp %s" % (user, key, ts))
     logging.debug("exception was: %s" % (e))
     get_client_stats_db_backup().insert(currEntry)
+
+  if success == None:
```

Comment:
where is the variable "success" defined? this should just fail immediately, right? Is there a test case for this?